packages/homepage/src/components/Toast/index.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState, useEffect } from 'react';
+import { Link } from 'gatsby';
+import { AnimatePresence } from 'framer-motion';
+
+import { ToastContainer } from './elements';
+
+const Privacy = () => {
```

Comment:
```suggestion
export const Privacy: FunctionComponent = () => {
```

Reply:
We are not using named exports or Typpescript in the homepage

Reply:
Doh my bad! 😓